google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -843,4 +852,20 @@ public void close() throws Exception {
   int getNumPendingWrites() {
     return pendingWrites.size();
   }
+
+  private static String createDefaultTimeRangeFilter() {
+    DateFormat rfcDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+    return "timestamp>=\"" + rfcDateFormat.format(yesterday()) + "\"";
+  }
+
+  private static Date yesterday() {
+    Calendar calendar = Calendar.getInstance();
+    calendar.add(Calendar.DATE, -1);
+    calendar.set(Calendar.HOUR_OF_DAY, 0);
+    calendar.set(Calendar.MINUTE, 0);
+    calendar.set(Calendar.SECOND, 0);
+    calendar.set(Calendar.MILLISECOND, 0);
```

Comment:
This reads to me like "yesterday at midnight", not "24 hours ago". Is that right?